### DISCUSSION TITLE
Not playing nicely.

### DISCUSSION THREAD

**Discussion by wookalarsc** [rg35xx-cfw/Koriki#113] [General]
I have tried the latest koriki "batocera-h5-rg35xx-plus-39-20240220.img" and had it boot and play fine, I tried installing better roms on it and removing the non english ones (replaced ctr crash team racing to fix the artifacts on screen) and it worked fine. I shot it down and the next day it would not turn on. I tried reset and the power button and made sure it was fully charged. yet it still will not boot. if I try the anbernic 64gig image it works after a quick resaet then power on, yet it will not boot to koriki now. I have tried 3 different sd cards ( none of them the original, it looked to cheap to rely on.  I would appreciate any suggestions on getting it up on koriki, it is so much smoother than the stock image. all of my sd cards are either Samsung or Sandisk 128 gig cards.

**Comment by italianbeef12** [comment]
This discussion forum is for users that have the 35XX. The image you are referring to is for the 35XX Plus, which is a different handheld. If you have a 35XX, you need to download this version: https://github.com/rg35xx-cfw/Koriki/releases/tag/koriki_batocera_rg35xx_20240105
